src/main/java/com/google/sps/servlets/DeleteHabitServlet.java

```diff
@@ -25,9 +25,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         KeyFactory keyFactory = datastore.newKeyFactory().setKind("HabitData");
         Key habitEntityKey = keyFactory.newKey(id);
 
-        System.out.println("before-habitEntityKey: " + habitEntityKey);
+        //System.out.println("before-habitEntityKey: " + habitEntityKey);
```

Comment:
nit: If you don't see yourself using these for debugging again, I would remove them for cleaner code.